### DISCUSSION TITLE
Error while runSofa on prompt and  there isn't the SOFA interface

### DISCUSSION THREAD

**Discussion by lowkey9527** [sofa-framework/sofa#4228] [Build / Config / Environment] [answered]
Hi,
when I run runSofa_d.exe I have this problem and there isn't SOFA interface

![Quicker_20231008_212440](https://github.com/sofa-framework/sofa/assets/125783921/6373fd8d-ddd1-46e9-a302-0b8933efa69d)

**Comment by alxbilger** [comment, marked as answer]
- You compiled SOFA in Debug mode. Is is what you wanted? If you are looking for performances, I recommend to compile in Release mode
- `runSofa` supports several GUIs, including no GUI. In your case, SOFA run the scene without GUI. The reason is that no other GUI was available. It's because the Qt-based GUI, stored in the `Sofa.GUI.Qt_d.dll` file has not been properly loaded. Unfortunately, the error message did not say why... I recommend to clear you build, and rebuild.

**Reply by lowkey9527** [reply]
@alxbilger Thank you so much. The problem has been solved
